### DISCUSSION TITLE
Can we able to change Form response format on event like submit or next ?

### DISCUSSION THREAD

**Discussion by venkateshpullaganti** [manojadams/mui-forms#63] [Q&A] [answered]
Hi @manojadams 

Is there a way that we can change the form response on submit / on next?
Currently, we are getting all the keys in a single object. Can we able to group the response?

**Comment by manojadams** [comment, marked as answer]
@venkateshpullaganti you can use the following approaches for custom grouping:
- https://mui-forms.vercel.app/fields/general-properties#2-example-with-prop
- https://mui-forms.vercel.app/fields/general-properties#3-example-with-nested-prop

You would need to modify `prop` property value for grouping.

**Reply by venkateshpullaganti** [reply]
Thank you manoj
